cli/src/commands/init.js

```diff
@@ -11,12 +11,13 @@ import inquirer from 'inquirer';
 import boxen from 'boxen';
 import { dockerComposeUp } from './services';
 import { uniqueNamesGenerator } from 'unique-names-generator';
-import { getServices, updateProjectFile, generateDockerCompose, failSpinner, startSpinner, succeedSpinner, verifySystem, consoleError, stopSpinner, getMissingImgs, getContainerNames } from '../utils';
+import { getServices, updateProjectFile, generateDockerCompose, failSpinner, startSpinner, succeedSpinner, verifySystem, consoleError, stopSpinner, getMissingImgs, getContainerNames, displayUpdateMessage } from '../utils';
 
 const access = promisify(fs.access);
 const copy = promisify(ncp);
 
 export async function initCommand(cmd) {
+    await displayUpdateMessage();
```

Comment:
Don't know if there's a way using commander js to call displayUpdateMessage first regardless of the command being executed.

Reply:
I don't think we need that. init and up should be sufficient

Reply:
ok